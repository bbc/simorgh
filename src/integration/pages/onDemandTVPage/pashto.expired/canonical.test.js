/**
 * @service pashto
 * @pathname /pashto/bbc_pashto_tv/tv/w172xcldhhrhmcf
 */

import runExpiredEpisodeTests from '../expiredEpisodeTests';
import runCommonEpisodeTests from '../commonEpisodeTests';
import {
  runCoreCanonicalTests,
  runCanonicalAnalyticsTests,
  runCommonCrossPlatformTests,
  runRecentEpisodesTests,
} from '../../../common';

describe('Canonical Pashto On Demand TV Page', () => {
  runExpiredEpisodeTests();
  runCoreCanonicalTests();
  runCanonicalAnalyticsTests();
  runCommonCrossPlatformTests(service);
  runCommonEpisodeTests();
  runRecentEpisodesTests();
});
