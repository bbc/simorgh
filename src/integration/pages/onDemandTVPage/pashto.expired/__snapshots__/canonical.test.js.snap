// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Pashto On Demand TV Page Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=68&amp;p=pashto.bbc_pashto_tv.tv.w172xcldhhrhmcf.page&amp;x1=[urn%3Abbc%3Apips%3Aw172xcldhhrhmcf]&amp;x2=[responsive]&amp;x3=[news-pashto]&amp;x4=[ps]&amp;x7=[player-episode]&amp;x8=[simorgh-nojs]&amp;x9=[%D9%86%DA%93%DB%8D%2520%D8%AF%D8%A7%2520%D9%88%D8%AE%D8%AA%2520-%2520BBC%2520News%2520%D9%BE%DA%9A%D8%AA%D9%88]\\"/>"`;

exports[`Canonical Pashto On Demand TV Page Expired Episode I can see the 'Content is not available' placeholder 1`] = `"دغه فایل نور د لاسرسي وړ نه دی."`;

exports[`Canonical Pashto On Demand TV Page Expired Episode I can see the brand title 1`] = `"نړۍ دا وخت"`;

exports[`Canonical Pashto On Demand TV Page Expired Episode I can see the episode summary 1`] = `"د بي بي سي پښتو ټلویزیوني خپرونه چې هره ورځ د افغانستان په شپږ بجو په ژوندۍ بڼه خپرېږي. دلته یې لیدلی شئ."`;

exports[`Canonical Pashto On Demand TV Page Expired Episode I can see the episode title 1`] = `"۲۸ می ۲۰۲۰"`;

exports[`Canonical Pashto On Demand TV Page Expired Episode a11y Assistive technology reads the brand and episode title as the headline 1`] = `"نړۍ دا وخت, ۲۸ می ۲۰۲۰"`;
